plugins/module_utils/common.py

```diff
@@ -26,12 +26,12 @@
 from datetime import datetime
 from distutils.version import LooseVersion
 
-
 from ansible.module_utils.basic import AnsibleModule, missing_required_lib
 from ansible.module_utils.six import iteritems, string_types
 from ansible.module_utils._text import to_native, to_bytes, to_text
 from ansible.module_utils.common.dict_transformations import dict_merge
 from ansible.module_utils.parsing.convert_bool import boolean
+from ansible_collections.community.kubernetes.plugins.module_utils.exceptions import K8sInventoryException
```

Comment:
Should it be from `kubernetes.core` instead of `community.kubernetes`?

Reply:
Yes. Thanks for correcting me.